src/otaclient/create_standby/delta_gen.py

```diff
@@ -572,6 +572,10 @@ def _calculate_delta(self, base_fst: str):
         for _input in self._fst_db_helper.iter_common_regular_entries_by_digest(
             base_fst
         ):
+            _digest, _ = _input
+            if _digest not in self._all_resource_digests:
+                continue  # skip resources that already prepared(via resume OTA)
+
```

Comment:
Could you add some comments regarding `_digest` and `_all_resource_digests`?
From the variable names, I was under the impression that _all_resource_digests might represent the set of digests that have been prepared(processed), like in the following logic:
```
            if _digest in self._all_resource_digests:
                continue 
```